.github/workflows/test-workflow.yml

```diff
@@ -47,7 +50,7 @@ jobs:
           version: nightly
 
       - name: Start the local node
-        run: npx hedera start -d --network local
+        run: npx hedera start -d --network-tag=${{inputs.networkTag}} --mirror-tag=${{inputs.mirrorTag}} --relay-tag=${{inputs.relayTag}} --verbose=trace
```

Comment:
Question: Above I saw the networkTag, mirrorTag, relayTag are marked required:false, does that mean if those tags are not entered, local node will pick up the latest tags?

Reply:
Nice work!

Reply:
My idea with this manual workflow is to be triggered, when for example we have a new network tag. Instead of checking out repos locally, starting local-node and running tests, we can do it here with a simple click.

Reply:
Nice initiative!